src/status_im/chat/handlers/send_message.cljs

```diff
@@ -179,22 +182,25 @@
 
 (register-handler ::prepare-message
   (u/side-effect!
-    (fn [db [_ {:keys [chat-id identity message] :as params}]]
-      (let [{:keys [group-chat clock-value]} (get-in db [:chats chat-id])
-            message'  (cu/check-author-direction
-                        db chat-id
-                        {:message-id   (random/id)
-                         :chat-id      chat-id
-                         :content      message
-                         :from         identity
-                         :content-type text-content-type
-                         :outgoing     true
-                         :timestamp    (time/now-ms)
-                         :clock-value  (inc clock-value)})
-            message'' (if group-chat
-                        (assoc message' :group-id chat-id :message-type :group-user-message)
-                        (assoc message' :to chat-id :message-type :user-message))
-            params'   (assoc params :message message'')]
+    (fn [{:keys [network-status]} [_ {:keys [chat-id identity message] :as params}]]
+      (let [{:keys [group-chat]} (get-in db [:chats chat-id])
```

Comment:
@alwx, I missed this when reviewed PR

> WARNING: Use of undeclared Var status-im.chat.handlers.send-message/db at line 186 /Users/rasom/clj/status-react/src/status_im/chat/handlers/send_message.cljs
WARNING: Use of undeclared Var status-im.chat.handlers.send-message/db at line 189 /Users/rasom/clj/status-react/src/status_im/chat/handlers/send_message.cljs